strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-rpm-layout-provider/src/test/java/org/carlspring/strongbox/artifact/coordinates/RpmArtifactCoordinatesTest.java

```diff
@@ -0,0 +1,99 @@
+            new ArrayList<>(Arrays.asList("libqxt-qt5", "0.7.0", "0.15.20130718giteaf6872f6ad4.fc30", "x86_64", "rpm")),
+            new ArrayList<>(Arrays.asList("atinject", "1", "28.20100611svn86.module_f28+3939+dc18cd75", "noarch", "rpm")),
+            new ArrayList<>(Arrays.asList("apache-commons-lang3", "3.7", "3.module_f28+3939+dc18cd75", "noarch", "rpm")),
+            new ArrayList<>(Arrays.asList("wingpanel-indicator-sound", "2.1.3", "1.fc30", "x86_64", "rpm")),
+            new ArrayList<>(Arrays.asList("libglvnd", "1.1.0", "4.gitf92208b.fc30", "i686", "rpm")),
+            new ArrayList<>(Arrays.asList("apache-commons-lang3", "3.7", "3.module_f28+3939+dc18cd75", "noarch", "rpm")),
+            new ArrayList<>(Arrays.asList("libglvnd", "1.1.0", "4.gitf92208b.fc30", "i686", "rpm")),
+            new ArrayList<>(Arrays.asList("totem", "3.32.1", "1.fc30", "x86_64", "rpm")),
+            new ArrayList<>(Arrays.asList("libsigc++20", "2.10.2", "1.fc30", "x86_64", "rpm")),
+            new ArrayList<>(Arrays.asList("egl-wayland", "1.1.3", "1.fc30", "x86_64", "rpm")),
+            new ArrayList<>(Arrays.asList("urw-base35-d050000l-fonts", "20170801", "12.fc30", "noarch", "rpm")),
+            new ArrayList<>(Arrays.asList("libbabeltrace", "1.5.6", "2.fc30", "x86_64", "rpm")),
+            new ArrayList<>(Arrays.asList("redhat-lsb-submod-security", "4.1", "47.fc30", "x86_64", "rpm")),
+            new ArrayList<>(Arrays.asList("cdi-api", "1.2", "8.module_f28+3939+dc18cd75", "noarch", "rpm"))));
+
+    private static List<String> packageExamples = Arrays.asList(
+            "libqxt-qt5-0.7.0-0.15.20130718giteaf6872f6ad4.fc30.x86_64.rpm",
+            "atinject-1-28.20100611svn86.module_f28+3939+dc18cd75.noarch.rpm",
+            "apache-commons-lang3-3.7-3.module_f28+3939+dc18cd75.noarch.rpm",
+            "wingpanel-indicator-sound-2.1.3-1.fc30.x86_64.rpm",
+            "libglvnd-1.1.0-4.gitf92208b.fc30.i686.rpm",
+            "apache-commons-lang3-3.7-3.module_f28+3939+dc18cd75.noarch.rpm",
+            "libglvnd-1.1.0-4.gitf92208b.fc30.i686.rpm",
+            "totem-3.32.1-1.fc30.x86_64.rpm",
+            "libsigc++20-2.10.2-1.fc30.x86_64.rpm",
+            "egl-wayland-1.1.3-1.fc30.x86_64.rpm",
+            "urw-base35-d050000l-fonts-20170801-12.fc30.noarch.rpm",
+            "libbabeltrace-1.5.6-2.fc30.x86_64.rpm",
+            "redhat-lsb-submod-security-4.1-47.fc30.x86_64.rpm",
+            "cdi-api-1.2-8.module_f28+3939+dc18cd75.noarch.rpm",
+            "libstdc++-9.2.1-1.fc30.src.rpm",
+            "gmp-c++-6.1.2-10.fc30.src.rpm");
+
+
+    @BeforeAll
+    private static void populatePackages()
+    {
+        parsedPackages.forEach(strings -> strings.add(4, RpmPackageType.BINARY.getPostfix()));
+
+        parsedPackages.add(new ArrayList<>(Arrays.asList("libstdc++", "9.2.1", "1.fc30", null, "src")));
+        parsedPackages.add(new ArrayList<>(Arrays.asList("gmp-c++", "6.1.2", "10.fc30", null, "src")));
```

Comment:
Please do everything what you need with `parsedPackages` in one place, either in the field declaration or in `@BeforeAll` method. I don't see the reason to split it into 2 places.

Reply:
Fixed. Is it ok now?